lightly_studio/src/lightly_studio/models/caption.py

```diff
@@ -57,17 +49,18 @@ class CaptionView(SQLModel):
     text: str
 
 
-class CaptionDetailsView(CaptionView):
-    """Response model for caption."""
+class CaptionViewsBySample(BaseModel):
+    """Response model for captions of a sample."""
 
-    sample: CaptionSampleView
+    sample_id: UUID
+    captions: List[CaptionView]
 
 
-class CaptionsListView(BaseModel):
+class CaptionViewsBySampleWithCount(BaseModel):
     """Response model for counted captions."""
 
     model_config = ConfigDict(populate_by_name=True)
 
-    captions: List[CaptionDetailsView] = PydanticField(..., alias="data")
+    samples: List[CaptionViewsBySample] = PydanticField(..., alias="data")
```

Comment:
Where do we construct CaptionViewsBySample?

Reply:
We've the class declaration [here](https://github.com/lightly-ai/lightly-studio/blob/1d87e4bcfad752028fc7cc789327b11045a33bf5/lightly_studio/src/lightly_studio/models/caption.py#L52).